CMake/options.cmake

```diff
@@ -194,7 +194,9 @@ set (HAVE_LIBPNG 1)
 if (UNIX)
   option (OPTION_CREATE_LINKS "create backwards compatibility links" OFF)
   list (APPEND FLTK_LDLIBS -lm)
-  option (OPTION_USE_WAYLAND "support Wayland and X11 backends" OFF)
+  if (NOT APPLE)
+    option (OPTION_USE_WAYLAND "support Wayland and X11 backends" ON)
```

Comment:
I wonder if this ought to read "support BOTH Wayland and X11 backends" or similar, to make it clearer that this is the hybrid case?

Reply:
Good suggestion. Will do.

Reply:
Done.